src/cfn/template.yaml

```diff
@@ -470,6 +780,11 @@ Resources:
       ServiceToken: !GetAtt LambdaSetup.Arn
       Region: !Ref AWS::Region
 
+  VPC:
```

Comment:
Can we move this to the top of the file? Also, we need `EnableDnsHostnames` set to true, you can see the options here:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-vpc.html#cfn-aws-ec2-vpc-EnableDnsSupport